{{cookiecutter.slug}}/frontend.angular/src/app/services/language.service.ts

```diff
@@ -0,0 +1,52 @@
+import { HttpClient, HttpErrorResponse } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { lastValueFrom } from 'rxjs';
+
+export interface LanguageInfo {
+    current: string;
+    supported: {
+        code: string,
+        name: string
+    }[];
+}
+
+@Injectable({
+    providedIn: 'root'
+})
+export class LanguageService {
+    baseApiUrl = '/api';
+
+    constructor(private http: HttpClient) {
+    }
+
+    async get(): Promise<LanguageInfo> {
+        const response = await lastValueFrom(this.http.get<{
```

Comment:
Out of interest: why all the `await`? This could be handles by Observable subscriptions.